components/gitpod-protocol/data/gitpod-schema.json

```diff
@@ -71,11 +71,11 @@
                     },
                     "before": {
                         "type": "string",
-                        "description": "A shell command to run before `init` and the main `command`. This command is executed on every start and is expected to terminate. If it fails, the following commands will not be executed."
```

Comment:
Why drop this important piece of info? Already helped me multiple times in recalling how the interactions works.

Even better: point to a docs page incl diagram, for instance: https://www.gitpod.io/docs/configure/workspaces/tasks#execution-order

Reply:
Thank you for noticing this 😄 

My main concern was this:
> If it fails, the following commands will not be executed.

I assume this is true when you nest `before`, `command`, `init` together in a task?

```yaml
tasks:
  - name: Docs Server
    before: cd docs
    init: npm install
    command: npm run dev
```

I think it's not valid when not nested:

```yaml
tasks:
  - name: random
    before: exit 1

  - name: npm
    command: npm install && npm run dev
```

So, would it be good if we put the docs link instead as you suggested 🙏 ?

Reply:
We could drop only the last sentence or make it clear that it's about the following items in the array.

Reply:
That also sounds like a good idea.